src/react/DateTime/index.tsx

```diff
@@ -208,28 +210,23 @@ const DateTime = ({
                   dateValue.setSeconds(timeDateValue.getSeconds());
                   dateValue.setMinutes(timeDateValue.getMinutes());
                   dateValue.setHours(timeDateValue.getHours());
+
                   setTimeDateValue(temporary);
                   setTimeout(() => {
                     setPickerDisplay('block');
                     setTimePickerDisplay('none');
                     setTimeDateValue(dateValue);
                   }, 1);
                 }}
-                value={inputValue}
+                value={value !== null ? inputValue : ''}
                 onBlur={(e) => {
                   const tempDate = parseStringToDate(e.target.value, dateFormat);
                   let returnDate: Date|null = null;
                   if (!e.target.value) {
-                    const todayDate = new Date();
-                    todayDate.setSeconds(0);
-                    todayDate.setHours(timeDateValue.getHours());
-                    todayDate.setMinutes(timeDateValue.getMinutes());
-                    if (todayDate.getTime() !== value.getTime()) {
-                      returnDate = new Date(todayDate);
-                    }
+                    returnDate = null;
```

Comment:
Please help me re check this logic, This line will be return null time value when on blur